local/docker-compose.yml

```diff
@@ -18,64 +19,66 @@ services:
     hostname: $PROJECT_NAME-mysql
     container_name: $PROJECT_NAME-mysql
     environment:
-      - MYSQL_DB=$MYSQL_DB
-      - MYSQL_USER=$MYSQL_USER
-      - MYSQL_PASS=$MYSQL_PASS
+      MYSQL_DB:
+      MYSQL_USER:
+      MYSQL_PASS:
     volumes:
       - "site-database:/var/lib/mysql:rw"
+      - "site-database-log:/var/log/mysql:rw"
     networks:
       - default
 
-  drupal:
+  site:
```

Comment:
Hmmmmmwince. I do call them all `drupal` for the drupal properties, only `site` for things that aren't drupal.

Reply:
Sorry for the wince! At the moment in the local stacks they are all called 'site', e.g. https://github.com/UN-OCHA/rwint9-site/blob/751cc4b056550f770e75390a3d0e9c0ab541a866/local/docker-compose.yml#L64 - I don't have a strong preference but would be good to have it consistent for all sites.